app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <title>ApprenticeshipStandardsDotOrg</title>
+    <title><%= @page_title ? @page_title + " - ApprenticeshipStandardsDotOrg" : "ApprenticeshipStandardsDotOrg" %></title>
```

Comment:
@jenlu33 if the controller does not define `@page_title`, this check will probably fail since the variable we are trying to access is not defined (need to verify with a test). We can do:

```
<%= defined?(@page_title) ? @page_title + " - ApprenticeshipStandardsDotOrg" : "ApprenticeshipStandardsDotOrg" %>
```

Reply:
@FerPerales if `@page_title` is not defined (is `nil`), then won't it just skip to the `else` part of the tertiary expression?